base/src/main/java/com/smartdevicelink/proxy/rpc/SystemCapability.java

```diff
@@ -37,6 +37,7 @@
 import com.smartdevicelink.proxy.rpc.enums.SystemCapabilityType;
 
```

Comment:
There are some spacing/formatting issues in this file. They are not introduced in this PR though. Could you please fix them in this PR as well since we are already making changes to the file?